nifi-nar-bundles/nifi-extension-utils/nifi-processor-utils/src/main/java/org/apache/nifi/processor/util/bin/BinFiles.java

```diff
@@ -94,6 +96,16 @@
             .addValidator(StandardValidators.createTimePeriodValidator(1, TimeUnit.SECONDS, Integer.MAX_VALUE, TimeUnit.SECONDS))
             .build();
 
+    public static final PropertyDescriptor ADD_MERGED_UUID = new PropertyDescriptor.Builder()
```

Comment:
I talked this over with @markap14 as well, it seems like just adding an attribute here is a better user experience than adding another property to offer a choice. I presume it was to offer the current behavior as well, which normally is a good thing so as not to upset existing flows, but I think in this case you could just add it (and document it). New attributes on split flowfiles might affect memory, but for merged flow files it should not be an issue. What do you think?